API/DTOs/OrderDTO.cs

```diff
@@ -7,7 +7,12 @@ public class OrderDTO
     {
         public Guid? Id { get; set; }
         [Required]
+        public Guid? CustomerId { get; set; }
```

Comment:
kadangi DTO yra tas objektas, kuri useris siuncia i backenda, tai manau nereikia customerId siusti, nes jis bus paiimamas is requesto headeriu, kai autentikacija bus. CustomerId reiktu idet i modeli, nes duombazej jau reik issaugot kuris customeris dare orderi